### ISSUE TITLE
no option "AirPlay Display" in Displays

### ISSUE THREAD

**Issue by grochib** [varszegimarcell/Optiplex-3020-Hackintosh-OpenCore#75]
I successfully installed first Big Sur and then Monterey (with the new EFI from tonymacx360) with you guidance on a 3020SFF (i5 4570S, 12GB RAM, no dGPU). 

Everything works great EXCEPT the missing option "AirPlay Display" in Display. Attached you can see from others out of this forum:
![image](https://user-images.githubusercontent.com/55094750/214965978-334d5557-cb5e-42a3-a7c9-3441650e6a76.png)

This is from my settings -> Display:

<img width="685" alt="Bildschirm­foto 2023-01-26 um 23 09 08" src="https://user-images.githubusercontent.com/55094750/214961852-a816e387-06fb-42aa-a523-63f05e4100cb.png">

Do you have any idea why there is no option?

Thanks a lot!

On a side note: the first time I installed MacOS the screen turned black at some point. After digging for a while I realized that it was an issue with the deviceProperties in config.plist. After deleting the PciRoot(0x0)/Pci(0x2,0x0) entry it booted without any issue. 
I could set up screen sharing so when adding the HD4600 to the DeviceProperties again I started the 3020SFF only to realize that Monterey started normally even if the screen stayd black. 
After a while I realized that this was an issue with my screen: when trying to change the input my onscreen menu was flickering. Turning the screen off and back on again solved it. 
Somehow the iGPU cannot wake up the screen or something like that. 
I use a DP to HDMI Adapter, quite an expensive one in order not to get some crappy adapter.

**Comment by Xtendera** [comment]
> I successfully installed first Big Sur and then Monterey (with the new EFI from tonymacx360) with you guidance on a 3020SFF (i5 4570S, 12GB RAM, no dGPU).
> 
> Everything works great EXCEPT the missing option "AirPlay Display" in Display. Attached you can see from others out of this forum: ![image](https://user-images.githubusercontent.com/55094750/214965978-334d5557-cb5e-42a3-a7c9-3441650e6a76.png)
> 
> This is from my settings -> Display:
> 
> <img alt="Bildschirm­foto 2023-01-26 um 23 09 08" width="685" src="https://user-images.githubusercontent.com/55094750/214961852-a816e387-06fb-42aa-a523-63f05e4100cb.png">
> 
> Do you have any idea why there is no option?
> 
> Thanks a lot!
> 
> On a side note: the first time I installed MacOS the screen turned black at some point. After digging for a while I realized that it was an issue with the deviceProperties in config.plist. After deleting the PciRoot(0x0)/Pci(0x2,0x0) entry it booted without any issue. I could set up screen sharing so when adding the HD4600 to the DeviceProperties again I started the 3020SFF only to realize that Monterey started normally even if the screen stayd black. After a while I realized that this was an issue with my screen: when trying to change the input my onscreen menu was flickering. Turning the screen off and back on again solved it. Somehow the iGPU cannot wake up the screen or something like that. I use a DP to HDMI Adapter, quite an expensive one in order not to get some crappy adapter.

Wish I could help. I have not done hackintosh in more than two years now. Sorry 😦 . Hope someone else can help.

**Comment by Scott-GT** [comment]
I don't think this is a 'screen/display' related issue and more WiFi/Bluetooth. Those will have to be setup perfectly for this to work (if at all possible).
